pkg/ibmvpc/examples/out/explain_out/GroupingExternalSG1_all_vpcs_explain.txt

```diff
@@ -4,10 +4,14 @@ Connectivity explanation between vsi1-ky and 161.26.0.0/8
 No connection between vsi1-ky[10.240.10.4] and Public Internet 161.0.0.0-161.25.255.255,161.27.0.0-161.255.255.255; connection blocked by egress
 External traffic via PublicGateway: public-gw-ky
 Egress: security group sg1-ky blocks connection; network ACL acl1-ky allows connection
+Path:
+	vsi1-ky[10.240.10.4] -> |
 ------------------------------------------------------------------------------------------------------------------------
 
 The following connection exists between vsi1-ky[10.240.10.4] and Public Internet 161.26.0.0/16: protocol: UDP
-External traffic via PublicGateway: public-gw-ky
-Egress: security group sg1-ky allows connection; network ACL acl1-ky allows connection
+Path:
+	vsi1-ky[10.240.10.4] -> security group sg1-ky -> subnet1-ky -> network ACL acl1-ky -> 
```

Comment:
ok, sounds good. can you add an example test to demonstrate this?

Reply:
I can try to do it with `clVer`, not sure it will be that trivial -  it will require bridging between the API outputs and the expected input - I recall there is some difference. I already opened an issue for it back in Dec https://github.com/np-guard/vpc-network-config-analyzer/issues/265. I'll do it - the entire issue - when I context switch to `clVer`.

Reply:
why do we need clver? can do something simpler by copying some SG from another example, and editing the config object JSON file?

Reply:
I'm not a fan of manually modifying computer generated files.
However, I managed to add another (existing) SG to one of the VSIs, and it works well (Also the analysis above the explainability - e.g. one of the rules in one of the SG was about the SG I added to another VSI and the rule changed smoothly, look at the test)

Reply:
looks good